packages/application/src/services.tsx

```diff
@@ -39,4 +39,33 @@ export class FrontendServices {
 
     return metadataResponse[namespace];
   }
+
+  static async postMetadata(namespace: string, requestBody: any): Promise<any> {
+    const metadataResponse: any = await RequestHandler.makePostRequest(
+      'elyra/metadata/' + namespace,
```

Comment:
maybe make `elyra/metadata` a variable in the class and just reference the variable in each of the requests